sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -6,291 +6,129 @@
-<SALTelemetry>
- <Subsystem>MTMount</Subsystem>
- <Version>3.2.0</Version>
- <Author>Ander Ansuategi</Author>
- <EFDB_Topic>MTMount_Azimuth_Cable_Wrap</EFDB_Topic>
+ <EFDB_Topic>MTMount_Camera_Cable_Wrap</EFDB_Topic>
      <item>
-          <EFDB_Name>ACW_Status</EFDB_Name>
-          <Description>ACW_Status</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>CCW_Status</EFDB_Name>
+          <Description>CCW_Status</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>ACW_Status_Drive_1</EFDB_Name>
-          <Description>ACW_Status_Drive_1</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>CCW_Status_Drive_1</EFDB_Name>
+          <Description>CCW_Status_Drive_1</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>ACW_Status_Drive_2</EFDB_Name>
-          <Description>ACW_Status_Drive_2</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>CCW_Status_Drive_2</EFDB_Name>
+          <Description>CCW_Status_Drive_2</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Current_Drive_1</EFDB_Name>
-          <Description>ACW_Current_Drive_1</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Current_Drive_2</EFDB_Name>
-          <Description>ACW_Current_Drive_2</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_AngleDif_Actual</EFDB_Name>
-          <Description>ACW_AngleDif_Actual</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_AngleDif_ActualLVDT</EFDB_Name>
-          <Description>ACW_AngleDif_ActualLVDT</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Angle_Setpoint</EFDB_Name>
-          <Description>ACW_Angle_Setpoint</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>deg</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Positive_Power_Off_switch</EFDB_Name>
-          <Description>ACW_Positive_Power_Off_switch</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```

Comment:
Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.